transform_aes.c

```diff
@@ -162,7 +189,7 @@ static ssize_t aes_choose_rx_sa( transop_aes_t * priv, const u_int8_t * peer_mac
  */
 static size_t aes_best_keysize(size_t numBytes)
 {
-    if (numBytes >= AES256_KEY_BYTES )
+    if (numBytes >= AES256_KEY_BYTES)
```

Comment:
That was my first thought, too. But to remain completely compatible to current builts (what _now_ is treated as V1), I suggest to keep it, otherwise compatibility of current dev version "-A" and this pull request's "-A1" will break.

As we saw in the benchmarks above, AES with 128 bit key is still 23% faster. So, my idea was, to leave it to the user to decide: by chosing the password length one determines the key length an thus computational load.

Based on explanation from #68, a 256 bit key is space is better than a 128 bit key space only. During the last days, when I found that the three zeroed keys (128, 192, 256 bits repspectively) seem to deliver different encryptions, I had the idea that using the full available keyspace (all 128, 192 **and** 256 bitted keys) would be more than just restricting to the full key size of 256 bit.

That is why I left this routine also for V2. An attacker would not know what size of key we have used, he would need to brute force all 128, 192 and 256 keys instead of all 256 bit keys only - which of course already are a lot.